htdocs/main.php

```diff
@@ -19,23 +19,26 @@
 set_include_path(get_include_path().":../project/libraries:../php/libraries:");
 require_once __DIR__ . "/../vendor/autoload.php";
 ini_set('default_charset', 'utf-8');
+
 ob_start('ob_gzhandler');
 // Create an output buffer to capture console output, separately from the
 // gzip handler.
 ob_start();
 
 // load the client
-$client = new NDB_Client;
-if ($client->initialize() == false) {
-    $login = $_SESSION['State']->getProperty('login');
-    $login->showLoginScreen();
-    return false;
+$client    = new NDB_Client;
+$anonymous = $client->initialize() === false;
+// The NDB_Client->initialize() prints a bunch of garbage, so flush the
```

Comment:
why not clean the garbage ?

Reply:
it's beyond the scope of this PR, and comes from deep within places like SinglePointLogin.

Reply:
I wanna put a thumbs up but the reactions are not working..